src/main/cpp/t07_quad2.cpp

```diff
@@ -21,5 +21,34 @@
 using namespace std;
 
 int t07_quad2() {
-
+    double a, b, c, x1, x2, D;
+    cin >> a >> b >> c;
+    D = b * b - 4 * a * c;
+    if (a != 0) {
+        if (D == 0) {
+            x1 = (-b + sqrt(D)) / (2 * a);
+            cout << 1 << " " << x1;
+        }
+        else if (D > 0) {
+            x1 = (-b + sqrt(D)) / (2 * a);
+            x2 = (-b - sqrt(D)) / (2 * a);
+            if (x1 < x2)
+                cout << 2 << " " << x1 << " " << x2;
+            else
+                cout << 2 << " " << x2 << " " << x1;
+        }
+        else
+            cout << 0;
+    }
+    else if(a==0 && b!=0){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Condition 'a==0' is always true](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4353647)